docs/src/Theory/Atmos/Aerosol-activation.md

```diff
@@ -0,0 +1,177 @@
+# Aerosol Activation Parametrization
```

Comment:
I think we should rename this file to `aerosol_activation.md`, so that it follows the convention we're using.